CONTRIBUTING.md

```diff
@@ -190,15 +190,15 @@ INPUT_SKIP_ITEMS=
 INPUT_DRY_RUN=true
 INPUT_SKIP_ASSEMBLE=false
 INPUT_SKIP_REGENERATE=false
-INPUT_REPOSITORY=.
+INPUT_REPO_PATH=.
 INPUT_BRANCH=test
-INPUT_MARKDOWN_PATH=markdown/profiles
+INPUT_MARKDOWN_DIR=markdown/profiles
 INPUT_OSCAL_MODEL=profile
-INPUT_SSP_INDEX_PATH=
+INPUT_SSP_INDEX_FILE=
```

Comment:
I believe these environment variables will need to have the `TRESTLEBOT` prefix.  E.x. `INPUT_TRESTLEBOT_SSP_INDEX_FILE=`